server/planning/events/events_schema.py

```diff
@@ -351,5 +351,31 @@
         'type': 'list',
         'required': False,
         'schema': {'type': 'string'}
-    }
+    },
+
+    # HACK: Add coverages.planning.slugline to elastic mapping
+    # Otherwise searching slugline in combined view fails on events type
+    'coverages': {
+        'type': 'list',
+        'mapping': {
+            'type': 'nested',
+            'properties': {
+                'planning': {
+                    'type': 'object',
```

Comment:
might be good to set this not dynamic to avoid indexing other fields

Reply:
Good point, will update this tomorrow.
I didn't like this hack, but the other alternative was to send two queries to elasticsearch.